styles.css

```
@@ -0,0 +1,439 @@
+  font-size: 2.5vw;
+}
+
+/* Contact */
+
+.contact_container {
+  text-align: center;
+}
+
+.form {
+  width: 80%;
+  margin: 0 auto;
+}
+
+fieldset {
+  margin: 2vw;
+  padding: 2vw;
+  border: 1px solid #fc66d1;
+}
+
+legend {
+  color: #66fc91;
+  font-size: 6vw;
+}
+
+label {
+  display: block;
+  color: #fc66d1;
+  font-size: 5vw;
+  text-align: center;
+  margin: 0 auto;
+}
+
+input {
+  display: block;
+  height: 7vw;
+  width: 80%;
+  margin: 3vw auto;
+  font-size: 4vw;
+}
+
+textarea {
+  display: block;
+  width: 90%;
+  height: 16vw;
+  margin: 3vw auto;
+  font-size: 4vw;
+}
+
+textarea::placeholder {
+  font-size: 4vw;
+}
+
+button {
+  color: #66fc91;
+  background-color: black;
+  border: 1px solid #fc66d1;
+  margin-top: 3vw;
+  padding: 2vw 3vw;
+  border-radius: 15%;
+}
+
+button:hover {
+  background-color: #fc66d1;
+}
+
+/* Footer */
+#footer {
+  color: #66fc91;
+  text-align: center;
+  padding: 5vw 0;
+  font-size: 3vw;
+}
+
+/*Media Query min-width 768px*/
+@media screen and (min-width: 768px) {
+  .form {
+    width: 90%;
+  }
+
+  legend {
+    font-size: 5vw;
+  }
+
+  label {
+    display: inline;
+    font-size: 4vw;
+    text-align: left;
+  }
+
+  input {
+    display: inline;
+  }
+
+  .contact_reason {
+    display: grid;
+    grid-template-columns: 1fr 3fr;
+    align-items: center;
+  }
+
+  input[type="radio"] {
+    height: 7vw;
+    width: 3vw;
+  }
+
+  textarea {
+    display: inline;
+    width: 90%;
+  }
+
+  textarea::placeholder {
+    font-size: 4vw;
+  }
+
+  button {
+    font-size: 3.8vw;
+  }
+
+  #footer {
+    font-size: 2.5vw;
+  }
+}
+
+/*Media Query min-width 1200px*/
+@media screen and (min-width: 1200px) {
+  /* Heading */
+  h1 {
+    font-size: 6vw;
+    margin: 2vw 0 0 0;
+  }
+
+  h3 {
+    font-size: 2vw;
+    margin: 0 2.7rem;
+  }
+
+  ul {
+    margin: 1vw 0 1.8vw 0;
+  }
+
+  a {
+    font-size: 4vw;
+    margin: 0 2vw;
+  }
+
+  .figure_sara {
+    display: block;
+    float: left;
+    width: 40%;
+    margin-left: 2vw;
+  }
+
+  /*Wanted paragraph next to photo, but column in mobile*/
+  .about_p {
+    position: relative;
+    top: -66vw;
+    margin: 0 4vw;
+    text-align: justify;
+    letter-spacing: 0.3vw;
+    font-size: 1.8vw;
+    width: 45%;
+    float: right;
+  }
+
+  figcaption {
+    font-size: 1.5vw;
+    margin-bottom: 3vw;
+  }
+
+  /* About Me */
+
+  h2 {
+    font-size: 4vw;
+    margin: 3vw auto;
+    padding: 2vw 1.5vw;
+    border: 0.7vw dashed #fc66d1;
+    border-radius: 15%;
+  }
+
+  .figure_container {
+    position: absolute;
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+    align-items: baseline;
+    margin: 1vw 2vw;
+    padding: 0;
+  }
+
+  .about_figure > figcaption {
+    padding: 0;
+  }
+
+  /*Projects*/
+  .projects_a {
+    margin: 0;
+  }
+
+  .projects_img {
+    margin: 0 auto;
+    width: 70%;
+  }
+
+  #img_portfolio {
+    object-fit: contain;
+  }
+
+  #img_highlands {
+    object-fit: contain;
+  }
```

You can define rules for two classes at once like so:
```suggestion
  #img_portfolio,
  #img_highlands {
    object-fit: contain;
  }
```